Code/Source/URDF/UrdfToFbxConverter.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <string>
```

Comment:
It would be better to use AZStd::string instead of std::string since the rest of the engine and components depend on it. (and AZStd:: in general)